PlaybackLab/SerializationDownloadsSampleApp/SerializationDownloadsSampleApp/Resources/Skin/skin-config/examples/tv-skin.json

```diff
@@ -0,0 +1,247 @@
+{
```

Comment:
Remove this file, no need for it. I would say remove the whole examples folder.

Reply:
Done